torchrl/data/__init__.py

```diff
@@ -46,3 +46,4 @@
     UnboundedContinuousTensorSpec,
     UnboundedDiscreteTensorSpec,
 )
+from .utils import check_no_exclusive_keys, consolidate_spec
```

Comment:
avoid local imports

Reply:
<img width="552" alt="Screenshot 2023-07-27 at 17 16 04" src="https://github.com/pytorch/rl/assets/55539777/6c28d4be-4acc-4319-9795-6b1dcb2e5537">
wait so this is also all wrong?

Reply:
Oh sorry in `__init__` it's ok nevermind
in other files I try to avoid them -- it crashes when running doctests